### PR TITLE
docs(timepicker): add demo for showSpinners property

### LINKED ISSUE #3574: docs(timepicker): add show spinners property demo
adding show spinners property demo(#3265)

### PR DESCRIPTION
**Pull request by tarusin**
adding demo with spinner property
close #3574

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+timepicker/demos/spinners/spinners.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-timepicker-spinners',
+  templateUrl: './spinners.html'
+})
+export class DemoTimepickerSpinnersComponent {
+  ismeridian = false;
+  allowSpinners = true;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+timepicker/demos/spinners/spinners.ts]: showSpinners

File: demo/src/app/components/+timepicker/demos/spinners/spinners.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-timepicker-spinners',
+  templateUrl: './spinners.html'
+})
+export class DemoTimepickerSpinnersComponent {
+  ismeridian = false;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+timepicker/demos/spinners/spinners.ts]: isMeridian

File: demo/src/app/components/+timepicker/demos/spinners/spinners.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  allowSpinners = true;
+  mytime: Date = new Date();
```
Comment (IlyaSurmay) [inline demo/src/app/components/+timepicker/demos/spinners/spinners.ts]: myTime / time

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**